<?xml version="1.0"?>
<launch>
    <node pkg="rtimulib_ros" type="rtimulib_ros" name="rtimulib_node" output="screen">
        <param name="calibration_file_path" value="$(find rtimulib_ros)/config"/>
        <param name="calibration_file_name" value="RTIMULib" />
        <param name="frame_id" value="map"/>
        <param name="topic_name" value="imu"/>
        <param name="update_rate" value="10"/>
    </node>
</launch>
